Marlin/ultralcd.cpp

```diff
@@ -2303,13 +2303,49 @@ void kill_screen(const char* lcd_msg) {
     }
 
     /**
+     * UBL Step-By-Step submenu
+     */
+    void _lcd_ubl_step_by_step() {
+      START_MENU();
+      MENU_BACK(MSG_UBL_LEVEL_BED);
+      MENU_ITEM(gcode, MSG_H1 " " MSG_UBL_BUILD_COLD_MESH, PSTR("G28\nG29 P1"));
+      MENU_ITEM(function, MSG_H2 " " MSG_UBL_SMART_FILLIN, _lcd_ubl_smart_fillin_cmd);
+      MENU_ITEM(submenu, MSG_H3 " " MSG_UBL_VALIDATE_MESH_MENU, _lcd_ubl_validate_mesh);
+      MENU_ITEM(gcode, MSG_H4 " " MSG_UBL_FINE_TUNE_ALL, PSTR("G29 P4 R999 T"));
+      MENU_ITEM(submenu, MSG_H5 " " MSG_UBL_VALIDATE_MESH_MENU, _lcd_ubl_validate_mesh);
+      MENU_ITEM(gcode, MSG_H6 " " MSG_UBL_FINE_TUNE_ALL, PSTR("G29 P4 R999 T"));
+      MENU_ITEM(function, MSG_H7 " " MSG_UBL_SAVE_MESH, _lcd_ubl_save_mesh_cmd);
```

Comment:
The H1-H5 messages are specifically just for numbers that would refer to a hotend (and not H1-H5 tags, as it were). For this submenu we can just use ordinal numbers and we don't need to add H6 and H7. I'll do a quick patch and merge this asap!